SEED-platform/seed
# Meter feature improvements

#### Any background context you want to provide?
NREL-side testing was done and feedback was given on potential improvements.

#### What's this PR do?
The following feature improvements are contained in this PR. The reasoning behind each change is also given:
- [x] Meter import modal is larger and columns are resizeable (#1919 item 2)
- [x] Meter detail page and notes detail pages show record address (#1919 item 4)
- [x] PM Property IDs shown during PM meter import process (#1919 item 1 and #1917)
- [x] PM Meter import supports workbook with multiple sheets (#1919 item 6)
- [x] Inventory detail meter page - sort list of meters to filter (this change came from my own testing)
- [x] Unknown Meter types and/or units are ignored - valid types and units are still displayed (#1918)

#### How should this be manually tested?
Go through the list above and ensure each has been updated and is working. 
Also, it might be useful to double check that other, non-meter related still work. I made some minor changes to the logic for reading Excel files.

For actual code review, it might be easier to analyze files by commit.

#### What are the relevant tickets?
#1917, #1918, #1919 

#### Screenshots (if appropriate)
